exercises/practice/simple-linked-list/.docs/instructions.append.md

```diff
@@ -1,4 +1,10 @@
 # Hints
 
-This exercise requires you to create a linked list data structure which can be iterated. This requires you to implement the IEnumerable\<T> interface.
-For more information, see [this page](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.ienumerable-1).
+This exercise requires you to create a linked list data structure which can be iterated. 
+
+1. Implement `Count` property - it should not be possible to change its value from the outside
+2. Implement `Push(T value)` method that adds a value to the list at its head. 
+3. Implement `Pop()` method which removes and returns a value from the head. 
+4. Add a constructor to allow initialisation with a single value, or with an interable
+5. Implement the `IEnumerable<T>` interface. For more information, see [this page](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.ienumerable-1).
+6. Ensure `Reverse()` method is available. 
```

Comment:
This is not something that students need to implement as they get it for free by implement IEnumerable, right?

Reply:
If they do it right. If they don't then the test will fail.